byol 5.1 release notes (#1062)
128technology/docs: docs/release_notes_byol_5.1.md

---
title: Bring Your Own License (BYOL)
sidebar_label: '5.1'
---
## Release 5.1.0

**Release Date:** June 30, 2026

#### New Features and Improvements

> - **I95-64995 Increase BYOL install timeouts and make configurable**

Reviewer: ```suggestion
- **I95-64995 Increase default BYOL install timeout and make it configurable**
```

Increased the default install timeout to 6 hours and allow for unlimited retries.

These values can be configured via cloud-init by setting the following options in the onboarding configuration.

> ```

Reviewer: this should be captured in the cloud-init sections for the corresponding BYOL docs for completeness. Otherwise it will just get lost in a release note as a foot note.

{
    "download-timeout": "10h",
    "download-retries": 5
}
```

### Resolved Issues

>  - **I95-65092 Packer leaves stale authorized key in the BYOL partition (AWS Only)**

Reviewer: ```suggestion
 - **I95-65092 Image creation leaves stale authorized key on the BYOL partition (AWS Only)**
```
